infrastructure/lib/lambda/PostConfirmationTriggerCognito.ts

```
@@ -26,10 +26,14 @@ export const handler = async (event: PostConfirmationTriggerEvent) => {
                     },
                     id: { S: userId },
                     username: { S: username },
+                    createdAt: { S: new Date().toISOString() },
+                    updatedAt: { S: new Date().toISOString() },
                 },
             }),
         );
     } catch (error: unknown) {
         console.error(error);
```

I think you want to rethrow the error